src/api/spec/features/webui/projects/label_template_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'browser_helper'
+
+RSpec.describe 'LabelTemplates', :js, :vcr do
+  let!(:user) { create(:confirmed_user, :with_home, login: 'Jane') }
+  let(:project) { user.home_project }
+
+  before do
+    Flipper.enable(:labels)
+
+    login user
+  end
+
+  context 'when having no label templates' do
+    it 'creates a label template' do
+      visit project_label_templates_path(project)
+
+      click_on('Create Label Template')
+      fill_in('Name', with: 'A label template')
+      click_on('Create')
+
+      expect(page).to have_text('Label template created successfully')
+    end
+  end
+
+  context 'when having an already existing label template' do
+    let(:label_template) { create(:label_template, project: project) }
+
+    before { label_template }
+
+    it 'updates an already existing label template' do
+      visit project_label_templates_path(project)
+
+      click_on('Edit')
+      fill_in('Name', with: 'A label template updated')
+      click_on('Update')
+
+      expect(page).to have_text('Label template updated successfully')
```

Comment:
I would test for the text of the LabelTemplate instead of the flash?

Reply:
Done